apps/sports-helsinki/src/domain/search/combinedSearch/adapters/VenueSearchAdapter.ts

```diff
@@ -33,10 +32,10 @@ class VenueSearchAdapter implements CombinedSearchAdapter<VenueSearchParams> {
   language: VenueSearchParams['language'];
   q: VenueSearchParams['q'];
   ontologyTreeIds: VenueSearchParams['ontologyWordIds'];
+  ontologyTreeIdsOrSet2: VenueSearchParams['ontologyTreeIdsOrSet2'];
```

Comment:
Just out of curiousity, why is it called "...set2"?

Reply:
For similarity with the LinkedEvents related keywordOrSet<X> names, although here I didn't rename ontologyTreeIds to ontologyTreeIdsOrSet1